base/shared/src/main/scala/scalaz/data/Kleisli.scala

```diff
@@ -0,0 +1,35 @@
+package scalaz
+package data
+
+sealed trait KleisliModule {
+  type Kleisli[F[_], A, B]
+
+  def runKleisli[F[_], A, B](k: Kleisli[F, A, B]): A => F[B]
+
+  def wrapKleisli[F[_], A, B](k: A => F[B]): Kleisli[F, A, B]
```

Comment:
I'm unsure if these need the `Kleisli` suffix but I think it's good either way.

Reply:
All names are up for debate 😄 - i'm not attached to any name expressing my type signatures

Reply:
is it worth having an Iso?

      def iso[F[_], A, B]: Iso[A => F[B], Kleisli[F, A, B]]

Reply:
I didn't even notice we had `Iso`!